tests/test_special_euclidean.py

```diff
@@ -735,8 +881,8 @@ def test_log_from_identity_left(self):
     @geomstats.tests.np_only
     def test_exp_then_log_from_identity_left(self):
         """
-        Test that the riemannian left exponential from the identity
-        and the riemannian left logarithm from the identity
+        Test that the Riemannian left exponential from the identity
```

Comment:
Thanks!

Reply:
Someone else is to be thanked for that, don't know who though.